pkg/builder/controller.go

```diff
@@ -138,26 +135,24 @@ func (blder *Builder) Named(name string) *Builder {
 // Complete builds the Application ControllerManagedBy.
 func (blder *Builder) Complete(r reconcile.Reconciler) error {
 	_, err := blder.Build(r)
-	return err
+	if err != nil {
```

Comment:
This change isn't necessary. The error is already being handled

Reply:
Reverted